ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -180,6 +181,18 @@ const routeComponentMap: Record<RouteKey, RouteComponent> = {
         ),
         path: vulnerabilitiesPlatformCvesPath,
     },
+    'platform-workload-cves': {
```

Comment:
Off the main subject of this contribution, when I compared to the pattern in `'workload-cves'` the question came to mind about pro and con to group the keys (which are internal) similar to the potential precedent of `'vulnerabilities/reports'`
* `'vulnerabilities/node-cves'`
* `'vulnerabilities/platform-cves'`
* `'vulnerabilities/platform-workload-cves'`
* `'vulnerabilities/workload-cves'`

To show that I am not a slave to patterns, I left out `'exception-configuration'` and `'exception-management'` as possibly clearer adjacent than separated.

Reply:
I wouldn't be opposed to this. I'd even say that `exception-management` would make sense here as well (but with `exception-configuration` separate at the top level).